vuepress/docs/next/tutorials/ecr/github-actions-workflow-from-ecg.md

```diff
@@ -0,0 +1,219 @@
+
+## Start generating a GitHub Action Workflow
+
+From your main project folder:
+
+1. Run `npm install` if needed
+2. Run the JHipster `ci-cd` subgenerator and select `GitHub Actions` (e.g. run `ent jhipster ci-cd` in the ent CLI) 
+3. Deselect all `tasks/integrations` to generate the default Workflow
+4. Check the YAML file content on `.github/workflows/github-ci.yml`
+
+``` yaml
+name: Application CI
+on: [ push, pull_request ]
+jobs:
+ pipeline:
+   name: ci pipeline
+   runs-on: ubuntu-latest
+   if: "!contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.pull_request.title, '[skip ci]') && !contains(github.event.pull_request.title, '[ci skip]')"
+   timeout-minutes: 40
+   env:
+     SPRING_OUTPUT_ANSI_ENABLED: DETECT
+     SPRING_JPA_SHOW_SQL: false
+   steps:
+     - uses: actions/checkout@v2
+     - uses: actions/setup-java@v1
+       with:
+         java-version: '11.x'
+     - name: Stop MySQL server
+       run: sudo /etc/init.d/mysql stop
+     - name: Run backend test
+       run: |
+         chmod +x mvnw
+         ./mvnw -ntp clean verify
+     - name: Package application
+       run: ./mvnw -ntp package -Pprod -DskipTests
+```
+
+## Refine the Workflow definition
+
+- Below are the key and the name of the Job, respectively:
+``` yaml
+ pipeline
+   name: ci pipeline
+```
+The key can be reused in the file as a reference, while the UI displays the Job using the name. By default, the name is defined as the key. For convenience, rename it to `backend`.
+
+- The Workflow is triggered each time a new commit is added or when a pull request is open:
+``` yaml
+on: [push, pull_request]
+```
+However, you can [customize the triggering events](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows). 
+
+- Remove the statement below to ensure the Workflow is not skipped:
+``` yaml
+ if: "!contains(github.event.head_commit.message, '[ci skip]') &&  
+ !contains(github.event.head_commit.message, '[skip ci]') && 
+ !contains(github.event.pull_request.title, '[skip ci]') && 
+ !contains(github.event.pull_request.title, '[ci skip]')"
+```
+GitHub [recently announced](https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/) a baseline feature where the Workflow is skipped if the pull request title or latest commit message contains either `skip ci` or `ci skip`.
+
+- To avoid conflicts when starting any other MySQL database (e.g. through containers), use the below action to stop the MySQL server pre-installed on the Ubuntu runners:
+``` yaml
+name: Stop MySQL server 
+```
+
+
+## Add configuration for micro frontends
+
+### Prerequisites
+For this part of the tutorial, you will need the code for both the backend and the micro frontends. The micro frontends can be generated with the ECG or created manually.
+
+### Start defining a new Job
```

Comment:
'Define a new Job' ?

Reply:
so the wording makes it seem like the Job is in progress and we're focused on getting things going, in which case to start doing something would be more precise. it's difficult to read between the lines when there's a bit of a language barrier and i'm not familiar with the content to the point i can determine nuances :/